drizzlepac/haputils/cell_utils.py

```diff
@@ -547,7 +548,12 @@ def find_corners(self, member='total'):
             fp = np.clip(self.footprint, 0, 1).astype(np.int16)
 
             # simple trick to remove noise and small regions 3x3 or less.
-            scmask = ndimage.binary_dilation(ndimage.binary_erosion(fp, iterations=3), iterations=2)
+            scmask_dilated_erroded = ndimage.binary_dilation(ndimage.binary_erosion(fp, iterations=3), iterations=2)
+            
+            # Start by smoothing out the edges of the chips/field
+            # this will remove rough edges up to 3 pixels deep along the image edge
+            scmask = ndimage.gaussian_filter(scmask_dilated_erroded.astype(np.float32) * 100, sigma=2) > 50
```

Comment:
Please add a much needed comment here as to the reason for multiplying by 100 (which leads to the "> 50" criterion).  I seem to recall this is scaling needed to give the actual values some presence over the weeds, but I may not recall correctly.  Hence, the need for a comment.

Reply:
Ahhh right. We could have removed it and used a threshold of 0.5, but warren wanted to keep it from being a decimal I think. I'll add a comment.